src/chrome/content/rules/Connect.me.xml

```diff
@@ -1,22 +1,16 @@
-<!--
-	^connect.me: Expired, missing certificate chain
-	www.connect.me: Refused
-
--->
 <ruleset name="Connect.me">
 
 	<!--	Complications:
 				-->
-  <target host="connect.me"/>
   <target host="www.connect.me"/>
 
 
 	<!--	Redirect keeps path and args,
 		but not forward slash:
 					-->
-	<rule from="^http://(?:www\.)?connect\.me/+"
-		to="https://www.respectnetwork.com/connect-me-2/" />
+	<rule from="^http:"
+		to="https:" />
 
-		<test url="http://connect.me//" />
+		<test url="http://www.connect.me/" />
```

Comment:
Should they just removed then? At the moment, we aren't needing any wildcards on this domain at this time.